DOCS: Update en Contributor Guides (#53)
personium/new-docs: src/document-writer/README.md

---
id: README
title: How to participate in Personium document maintenance and improvement
sidebar_label: How to participate in Personium document maintenance and improvement
---

The open source project Personium welcomes not only code but also documentation contributions.
If you can help improve the documentation, please follow the steps below.

> __Please prepare a GitHub account__

## Document improvements pointed out

> 1. Create improvement issue in issues on GitHub repository

Reviewer: ```suggestion
1. Create issue (label: bug or documentation) in GitHub repository
```

    1. [Personium Documentation Website issues](https://github.com/personium/website/issues)

> By performing the above, Personium project members or other participants can correct and publish based on the pointed out.

Reviewer: ```suggestion
Personium project members or other participants will then review and respond to the issue.
```

## Adding and updating documentation

> 1. Fork the GitHub repository

Reviewer: ```suggestion
1. Fork the following GitHub repository
```

    1. [Personium Documentation Website](https://github.com/personium/website)

> 1. Clone to Local repository

Reviewer: ```suggestion
1. Clone to your local environment
```

> 1. Update the document by referring to [Repository README](https://github.com/personium/website)

Reviewer: ```suggestion
1. Read the [README](https://github.com/personium/website) before you starts working on the bug fix and/or improvement.  
```

> 1. Push to Fork's own GitHub repository and issue a Pull Request to the parent repository of the Fork

Reviewer: ```suggestion
1. Push to your forked GitHub repository and send a Pull Request to the original repository
```

> By performing the above procedure, the Personium project members will review and publish it in an appropriate form.

Reviewer: ```suggestion
Personium project members will review, merge and publish it accordingly.
```
